src/table/utils/__tests__/selections-utils.spec.ts

```diff
@@ -70,18 +70,18 @@ describe('selections-utils', () => {
 
         const newState = reducer(state, action);
         expect(newState).toEqual({ ...state, rows: { [cell.qElemNumber]: cell.rowIdx }, colIdx: cell.colIdx });
-        expect(state.api.begin).toHaveBeenCalledTimes(1);
-        expect(state.api.select).toHaveBeenCalledWith({ method: 'selectHyperCubeCells', params });
-        expect(state.api.cancel).not.toHaveBeenCalled();
+        expect(state.api?.begin).toHaveBeenCalledTimes(1);
```

Comment:
Here you have defined state.api so it should always be defined. Does lint complain or something?

Reply:
As SelectionState.api is optional